proto/config.go

```diff
@@ -209,3 +209,22 @@ func (rs ReplicaSlice) MoveToFront(i int) {
 	copy(rs[1:i+1], rs[0:i])
 	rs[0] = front
 }
+
+// PercentAvail computes the percentage of disk space that is available.
+func (sc StoreCapacity) PercentAvail() float64 {
+	return float64(sc.Available) / float64(sc.Capacity)
```

Comment:
This is just moving the code from Store.go.  So it's been like this for quite awhile.
Either Panic or I think returning a 0 here might work too.  I can add either.